File: python/ex058.py
# Faça um algoritmo que leia a idade de vários alunos de uma turma. O programa
# vai parar quando for digitada a idade 999. No final, mostre quantos alunos
# existem na turma e qual é a média de idade do grupo.


ageStudent = 0
totalStudents = 0
avgAge = 0
sumAge = 0

while True:
    ageStudent =  int(input("Digite sua idade: "))
    totalStudents +=1
    sumAge += ageStudent
    if(ageStudent != 999):
        avgAge = sumAge / totalStudents
    else:
        break
    
print(f"A turma possui: {totalStudents} Alunos \n a media de idade é: {avgAge}")

File: csharp/ex058.cs
// Faça um algoritmo que leia a idade de vários alunos de uma turma. O programa
// vai parar quando for digitada a idade 999. No final, mostre quantos alunos
// existem na turma e qual é a média de idade do grupo.


class Ex058
{
  public int counter = 0;
  public int avgAgeStudents = 0;

  public int totalStudents;

  public int countAgeStudents;

  public void FirstCountdown()
  {
    while (true)
    {
      Console.WriteLine($"Digite uma idade para o aluno [{counter}]: ");
      int inputAge = Convert.ToInt32(Console.ReadLine());

      totalStudents += 1;
      countAgeStudents += inputAge;

      if (inputAge == 999)
      {
        break;
      }

      avgAgeStudents = countAgeStudents / totalStudents;
    }
    Console.Write($"Media de idade dos alunos da turma : {avgAgeStudents}");
  }
}

